src/content/docs/dropins-b2b/purchase-order/dictionary.mdx

```diff
@@ -0,0 +1,366 @@
+            "grand_total": "Grand Total",
+            "shipping_incl_tax": "Shipping Cost",
+            "number_of_skus": "Number of SKUs"
+          },
+          "operator": {
+            "more_than": "Is more than",
+            "less_than": "Is less than",
+            "more_than_or_equal_to": "Is more than or equal to",
+            "less_than_or_equal_to": "Is less than or equal to"
+          }
+        }
+      }
+    },
+    "historyLog": {
+      "headerText": "Purchase order history log",
+      "statusTitle": "Status Changes",
+      "emptyText": "No history log available.",
+      "status": {
+        "cancel": "Cancelled on {{date}}",
+        "reject": "Rejected on {{date}}",
+        "place_order_fail": "Failed to place order on {{date}}",
+        "apply_rules": "Rule applied on {{date}}",
+        "place_order": "Order placed on {{date}}",
+        "auto_approve": "Auto approved on {{date}}",
+        "approve": "Approved on {{date}}",
+        "submit": "Submitted for approval on {{date}}"
+      },
+      "buttons": {
+        "viewMore": "View More",
+        "viewLess": "View Less"
+      },
+      "ariaLabel": {
+        "showMore": "Show more history items",
+        "showLess": "Show fewer history items"
+      }
+    },
+    "comments": {
+      "view": {
+        "headerText": "Purchase order comments",
+        "emptyText": "No comments available.",
+        "buttons": {
+          "viewMore": "View More",
+          "viewLess": "View Less"
+        },
+        "ariaLabel": {
+          "showMore": "Show more comments",
+          "showLess": "Show fewer comments"
+        }
+      },
+      "add": {
+        "headerText": "Add purchase order comment",
+        "placeholder": "Add your comment",
+        "submit": "Add Comment",
+        "errorMessage": "Something went wrong while adding your comment. Please try again."
+      }
+    },
+    "approvalFlow": {
+      "headerText": "Purchase order approval flow",
+      "emptyText": "No approval flow is available for this purchase order.",
+      "ariaLabels": {
+        "icons": {
+          "approved": "Status approved",
+          "rejected": "Status rejected",
+          "pending": "Status pending approval"
+        }
+      }
+    }
+  }
+}
+```
+
+## How to customize
+
+You can override any of these dictionary values by creating a custom dictionary file and loading it during drop-in initialization. The drop-in will automatically deep-merge your custom values with these defaults.
+
+**For complete step-by-step instructions**, see the [Dictionary customization guide](/dropins/all/dictionaries/).
+
+**Quick example:**
+
+```javascript
+import { initialize } from '@dropins/storefront-purchase-order';
+
+await initialize({
+  langDefinitions: {
+    en_US: {
+  "PurchaseOrders": {
+    "customerPurchaseOrders": {
+      "containerTitle": "Custom string",
+      "noPurchaseOrders": "Custom string"
+    }
+  }
+}
+  }
+});
+```
+
+<Aside type="tip">
```

Comment:
This shouldn't be at the very bottom. Move to the general info at the top of the topic, or at least before the quick example